src/react-components/2d-hud.css

```diff
@@ -37,6 +37,8 @@
   display: flex;
   width: 48px;
   height: 48px;
+  -webkit-mask: url(../assets/hud/unmuted.png);
```

Comment:
Why is this in here? Does your editor autoprefix or is this a bug fix?

Reply:
bugfix, I noted it in the PR description.